HolyshiftService/JwtService.cs

```diff
@@ -0,0 +1,96 @@
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Security.Cryptography;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+
+public class JwtService
+{
+    private readonly byte[] _keyBytes = new byte[32];
+    private readonly string _issuer;
+    private readonly string _audience;
+    private readonly int _accessTokenExpirationMinutes;
+    private readonly int _refreshTokenExpirationDays;
+    private readonly ILogger<JwtService> _logger;
+
+    public JwtService(IConfiguration configuration, ILogger<JwtService> logger)
+    {
+        var jwtSettings = configuration.GetSection("JwtSettings");
```

Comment:
Note: In general avoid using void when it is not clear what is the type of the object.

examples:
- `var x = 1.ToString()` this is obvious a string so you can use var
- `for(var user in userList)` not very clear I might put the type. Unless `userList` type is clear in the previous line so I can deduce `user` type by reading
- `var x = GetRandomDataFromDb()` this is not clear what is the return type so specify the type.